LiteCore/Database/LiveQuerier.cc

```diff
@@ -43,7 +43,7 @@ namespace litecore {
                              Query *query,
                              bool continuous,
                              Delegate *delegate)
-    :Logging(QueryLog)
+    :Actor(QueryLog)
```

Comment:
Which part are you confused about?  This is simply a call to the base class constructor (since this class no longer inherits from Logging, it cannot call Logging)

Reply:
I got that part.  It used to be a Logger, now it is an Actor.  Why did it get such an amazing promotion?

Reply:
It always was an actor, it was just using the default constructor of Actor before.  C++ multiple inheritance can get very much into the weeds.....

Reply:
Oh!  Oh!  Of course...